scripts/utils/sign_and_send.js

```diff
@@ -41,10 +41,10 @@ module.exports = function (web3 = web3, artifacts = artifacts) {
       ADDRESS_0,
       nonce
     )
-    const lightWallet = await createLightwallet()
-    const account = lightWallet.accounts[0]
-    console.log(`Signing and posting multi-send transaction request from proposer account ${account}`)
-    const sigs = signTransaction(lightWallet, [account], transactionHash)
+    const privateKey = process.env.PK
+    const account = web3.eth.accounts.privateKeyToAccount("0x" + privateKey)
```

Comment:
Implementation question: does this line change the output of `web3.eth.getAccounts()`?

Reply:
Also seems hackey to be doing string manipulation in the vicinity of a private key...

Reply:
According to the documentation, is only creating an account object, which I am storing in the account variable:
https://web3js.readthedocs.io/en/v1.2.0/web3-eth-accounts.html#privatekeytoaccount

Reply:
@bh2smith  How would you propose to do it better?

Reply:
I guess this is standard, 

https://github.com/ethereum/web3.js/issues/1072#issuecomment-365184511

However, I wonder what would happen if I provided the "0x" along with the private when exporting it. For example, if I had copied my PK from ganache?

<img width="517" alt="Screenshot 2020-05-05 at 4 29 10 PM" src="https://user-images.githubusercontent.com/11778116/81077686-69dda100-8f41-11ea-9945-ab86760b2014.png">

which includes the 0x.